src/engraving/dom/paste.cpp

```diff
@@ -398,6 +398,7 @@ std::vector<EngravingItem*> Score::cmdPaste(const IMimeData* ms, MuseScoreView*
             addRefresh(target->abbox());         // layout() ?!
             EditData ddata(view);
             ddata.dropElement = el.get();
+            ddata.pos = target->abbox().topLeft();
```

Comment:
Have tested all element types to confirm only pasting onto symbols was currently making use of data.pos